worlds/adventure/Regions.py

```diff
@@ -154,6 +153,6 @@ def create_regions(options: PerGameCommonOptions, multiworld: MultiWorld, player
 
 
 # Placeholder for adding sets of priority locations at generation, possibly as an option in the future
-def determine_priority_locations(world: MultiWorld, dragon_slay_check: bool) -> {}:
+def determine_priority_locations() -> {}:
```

Comment:
on the one hand, this is a placeholder so maybe add the parameters to the comment or something
on the other hand, world: MultiWorld is cringe and should be deleted anyway

Reply:
I put the old function def in comments with `world: MultiWorld` changed